ceph_diagnostics_collect.sh

```diff
@@ -472,6 +473,21 @@ get_orch_info() {
     store    ${t}-host ${CEPH} orch host ls
 }
 
+get_prometheus_info() {
+    local t=prometheus_info
+    local target
+
+    info "collecting prometheus info ..."
+
+    store    ${t}-healthcheck_history_ls ${CEPH} healthcheck history ls
+    store -s ${t}-file_sd_config         ${CEPH} prometheus file_sd_config
+
+    show_stored ${t}-file_sd_config | jq -r '.[].targets[]' |
```

Comment:
Some clients don't have `jq` installed which affects the collection process. Is there a workaround to replace it?

Reply:
Then we have already had a problem. Because the script already depends on jq to be install to censor sensitive data in json format (like passwords in `ceph config dump` output). And that is more serious problem then the current case.

I can imagine the following solutions/workarounds:

1) Check `jq` is installed at the beginning and just fail if it does not (with a useful error message to install jq or use a host where jq is installed).
2) Skip all commands that require `jq` if `jq` is not installed.
3) Replace `jq` with python one liners (python is more likely to be present).

I prefer (1): usually it is not a problem to install `jq`, or one may use `cephadm shell` to run the script inside.